src/App.css

```diff
@@ -1,9 +1,209 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
 .App-header {
   background-color: #e24e54;
   padding: 20px;
   color: white;
+  display: flex;
+  height: 100px;
+  flex-direction: row;
+  align-content: center;
+}
+
+.app-icon {
+  align-items: center;
+  flex-basis: 10%;
+  border: 1px solid black;
+}
+
+.app-search {
+  display: flex;
+  align-items: center;
+  flex-basis: 50%;
+}
+
+.app-signin {
+}
+
+.app-login {
 }
 
 .App-intro {
   font-size: large;
 }
+
+h2 {
+  font-size: 40px;
+  line-height: 47px;
+}
+
+p {
+  font-size: 14px;
+  line-height: 16px;
+  margin-bottom: 40px;
+}
+
+a {
+  color: #e24e54;
+}
+
+.container {
+  flex-direction: column;
+  background: #eee;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.contributed {
+  padding: 5px;
+  border-right: 1px solid gray;
+}
+
+.contributed1 {
+  padding: 5px;
+  border-right: 1px solid gray;
+}
+
+.contributed2 {
+  padding: 5px;
+}
+
+.hours-contributed {
+  word-spacing: 1px;
+  text-align: center;
+  font-size: 46px;
+  height: 46px;
+  color: #e24e54;
+}
+
+.projects-completed {
+  word-spacing: 1px;
+  font-size: 46px;
+  height: 46px;
+  color: #4ecae2;
+}
+
+.meetups {
+  word-spacing: 1px;
+  text-align: center;
+  font-size: 46px;
+  height: 46px;
+  color: #e24e54;
+}
+
+.number {
+  font-style: bold;
+}
+
+.details {
+  color: #828282;
+  font-style: italic;
+}
+
+.contributions-nested {
+  display: flex;
+}
+
+.container-nested {
+  overflow: auto;
+  align-content: center;
+}
+
+.contact {
+  line-height: 40px;
+}
+
+.card {
+  display: flex;
+  background: white;
+  margin: 0 auto;
+  margin-bottom: 25px;
+  width: 90%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  height: 100%;
+  padding: 20px;
+  align-items: center;
+}
+
+.pic {
+  justify-content: center;
+  display: flex;
+  height: 400px;
+  flex-basis: 20%;
+  margin: 20px;
+}
+
+img {
```

Comment:
We definitely don't want this applied to all images. In the world of web applications, I consider using html tag selectors in css directly generally bad practice as it can make the ability to track down css styles difficult once there starts to be lots of different styles

Reply:
Actually, I thought about this some more, and I suppose you'll want good sane defaults for basic things (e.g. headers, paragraphs, etc) defined directly with html tag selectors. I just don't think we want oval pictures to be the default style.